packages/expo-store-review/src/StoreReview.ts

```diff
@@ -8,8 +8,8 @@ import StoreReview from './ExpoStoreReview';
  * iOS 10.3 or greater
  * `SKStoreReviewController` class is available
  */
-export function isSupported(): boolean {
```

Comment:
Could you keep this method and throw a deprecation error informing people to use the new method `isAvailableAsync()`.

Reply:
Should I keep this method on `.web` too, if so, should it return false or just a warning too ? Anyway for now I'm keeping it only on this file.